### ISSUE TITLE
Build on Windows

### ISSUE THREAD

**Issue by ROLLDOCK** [subgraph/Vega#150]
Hello every body, I'm a student, I'm beginner in Java.
Now I want to build  Vega on Windows but I can't. I use helpful [https://github.com/subgraph/Vega/wiki/Building-Vega](url).
Can you help me, thank you very much.

**Comment by leisipeisi** [comment]
Could you post some more detailed information on your attempts an what the errors are / why you are not able to built Vega?
Maybe #147 might help you.

**Comment by ROLLDOCK** [comment]
yes, thank you verymuch. But no I can't export code to excute file .jar. Can you help me, please !

**Comment by leisipeisi** [comment]
Sorry, but if you can't tell me what your exact problem is, I am not able to help you.

**Comment by ROLLDOCK** [comment]
Oh, I'm sorry. I can't find the "main.java"  for export to an file excute .jar. I'm use eclipse to build it.

**Comment by leisipeisi** [comment]
Seems I missunderstood you, sorry, I thought you wanted to run it on Windows, not to create a .jar. I have never done this so I don't have any advice on this, sorry. Maybe someone else might help you?
